backend/src/exceptions/invalid-credentials-error/invalid-credentials-error.exception.ts

```diff
@@ -0,0 +1,10 @@
+import { CustomExceptionName, ValidationMessage } from '~/common/enums/enums';
+
+class InvalidCredentials extends Error {
```

Comment:
```suggestion
class AuthError extends HttpError {
```
the error should be per service. also, extends from httperror